docs/faq.rst

```diff
@@ -104,8 +126,27 @@ You must fetch the channel directly and then call the appropriate method. Exampl
     channel = client.get_channel(12324234183172)
     await channel.send('hello')
 
+How do I send a DM?
+~~~~~~~~~~~~~~~~~~~
+
+Get the :class:`User` object and call :meth:`User.send`. For example: ::
+
+	user = client.get_user(381870129706958858)
+	await user.send('👀')
```

Comment:
Also needs an example showing `message.author.send` since that's 99.99999% the more common use case.